### ISSUE TITLE
Currently

### ISSUE THREAD

**Issue by pharringtonp19** [pharringtonp19/rfp#53]
- [ ] [inner map](https://github.com/pharringtonp19/rfp/blob/main/notebooks-scripts/motivation/inner_map.ipynb)
  - [ ] Explore inner loop loss fn 
- [ ] generate simulation data
- [ ] Implement bert standard training
    - [ ] tokenization (CURRENTLY WORKING ON)
- [ ] implement standard training
   - [x] Basic Setup 
      - [ ] Batching
      - [ ] Allow for `jax.nn.normalize`
      - [ ] Output shape of model
      - [x] treatment effect > on validation set
      - [x]  Fix `training_loss > validation_loss`
      - [x]  registering `model_params` with jax
   - [x] Add Plotting Section
   - [x] Treatment Heterogeneity
   - [x] Treatment Heterogeneity by other outcome variable
- [ ] implement rfp
   - [x] Basic Setup 
   - [ ] Add Plotting Section
   - [ ] Treatment Heterogeneity
   - [ ] Treatment Heterogeneity by other variable
